src/BuildingBlocks/Caching/Masa.BuildingBlocks.Caching/Default/DistributedCacheClientFactoryBase.cs

```diff
@@ -1,16 +1,16 @@
-﻿// Copyright (c) MASA Stack All rights reserved.
+// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
 namespace Masa.BuildingBlocks.Caching;
 
-public class DistributedCacheClientFactoryBase : CacheClientFactoryBase<IDistributedCacheClient>, IDistributedCacheClientFactory
+public class DistributedCacheClientFactoryBase : CacheClientFactoryBase<IDistributedCacheClientDisposeWrapper>, IDistributedCacheClientFactory
```

Comment:
IDistributedCacheClientDisposeWrapper? IFullDistributedCacheClient? IManualDistributedCacheClient?
@MapleWithoutWords @zhenlei520

Reply:
Which one can be released？IFullDistributedCacheClient？

Reply:
i think not very intuitive. @doddgu

Reply:
To consider the extension, the future only dispose for it?

Reply:
use IManualDistributedCacheClient